manage/update-px-rack.md

```diff
@@ -17,128 +17,40 @@ If PX nodes are provided with the information about their racks then they can us
-[root@vm-1 ~]#
-
-```
-kubectl will show the node vm-2 with the new ``px/rack`` label.
-
-Double check  if the rack information is reflected in the PX cluster.
-
-```
-$ /opt/pwx/bin/pxctl cluster provision-status
-NODE        NODE STATUS        POOL        POOL STATUS .....   ZONE           REGION        RACK
-vm-2        Online                0        Online      .....   default        default       rack1
-vm-3        Online                0        Online      .....   default        default       default
-
-```
-The node vm-2 which was labelled ``rack1`` is reflected on the PX node while the unlabelled node vm-3 is still using the ``default`` rack info.
-
-All the subsequent updates to the node labels will be automatically picked up by the PX nodes. A deletion of a ``px/rack`` label will also be reflected.
-
-If kubernetes node labels is not the preferred way of updating rack information then environment variables could be used as explained in the next section.
+To update the rack information in Kubernetes using node labels refer [Update Rack Info in Kubernetes](https://docs.portworx.com/scheduler/kubernetes/update-px-rack.html)
 
 ### Updating rack information through environment variables
 
-Portworx can be made aware of its rack information through ``PWX_RACK`` environment variable. These environment variables can be provided through
+Portworx can be made aware of its rack information through ``PWX_RACK`` environment variable. This environment variable can be provided through
 the ```/etc/pwx/px_env``` file. A sample file looks like this
 
 ```
 # PX Environment File
-# Add variables in the following format to export into PX container
-# export PWX_EXAMPLE_VAR=foobar
+# Add variables in the following format to automatically export them into PX container
+# PWX_EXAMPLE_VAR=foobar
 PWX_RACK=rack3
 ```
 
 Add the ```PWX_RACK=<rack-id>``` entry to the end of this file and bounce the PX container. On every PX restart, all the variables defined in ``/etc/pwx/px_env`` will be exported as environment variables in the PX container. Please make sure the label is a string not starting with a special character or a number.
 
->**Note:** this method requires a reboot of the PX container.
-
-### Specifying rack placement for volumes in the storage class
-
-Once the nodes are updated with rack info you can specify how the volume data can spread across your different racks. Following is an example of a storage class that replicates its volume data across racks `rack1` and `rack2`
-
-```yaml
-##### Portworx storage class
-kind: StorageClass
-apiVersion: storage.k8s.io/v1beta1
-metadata:
-    name: px-postgres-sc
-provisioner: kubernetes.io/portworx-volume
-parameters:
-   repl: "2"
-   shared: "true"
-   racks: "rack1,rack2"
-```
-
-Any PVC created using the above storage class will have a replication factor of 2 and will have one copy of its data on `rack1` and the other copy on `rack2`
+>**Note:** This method requires a reboot of the PX container.
```

Comment:
The note should be at the top, right after the heading "Updating rack information through environment variables".
Similar for the section below